docs/content/en/docs/migrate/metrics-observe/_index.md

```diff
@@ -2,51 +2,335 @@
   resources.
-* Keptn v1 calculations that use weighting and scoring
-  cannot currently be implemented for Keptn
-  but this functionality is under development.
 
-For more information about working with Keptn metrics, see the
-[Keptn Metrics](../../implementing/evaluatemetrics.md)
-page.
+## Convert Keptn v1 SLIs/SLOs to Analysis resources
+
+The Keptn Analysis feature provides capabilities
+similar to those of the Keptn v1
+[Quality Gates](https://keptn.sh/docs/1.0.x/define/quality-gates/)
+feature
+but it uses Kubernetes resources to define the analysis to be done
+rather than the configuration files used for Keptn v1.
+Tools are provided to convert Keptn v1 SLI and SLO definitions
+to Keptn Analysis resources.
+
+These instructions assume that the same SLO file
+has been used for all services in the project,
+so only one `AnalysisDefinition` resource
+(named `my-project-ad`) is created.
+If your Keptn v1 project has multiple SLOs,
+you need to create a separate `AnalysisDefinition`
+with a unique namefor each one.
+
+The process is:
+
+1. If necessary, rename the SLI to a Kubernetes compatible name
+
+   Keptn requires that all SLI names adhere to the
+   [Kubernetes resource naming rules](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/).
+   The conversion tools preserve the name of each SLI
+   so you may need to modify your
+   [sli.yaml](https://keptn.sh/docs/1.0.x/reference/files/sli/)
+   and
+   [slo.yaml](https://keptn.sh/docs/1.0.x/reference/files/slo/)
+   files to use a name that matches the Kubernetes requirements.
+   For example, the name of the `Response_Time` SLI
+   should be changed to `response-time` in both files.
+
+   We plan to eventually enable the conversion tools
+   to automatically convert the SLI names; see
+   [Issue 2098](https://github.com/keptn/lifecycle-toolkit/issues/2098/).
+
+1. Convert the SLIs to AnalysisValueTemplates
+
+   The following command sequence converts
+   a Keptn v1 `sli.yaml` file to a Keptn
+   [AnalysisValueTemplate](../../yaml-crd-ref/analysisvaluetemplate.md)
+   resource:
+
+   ```shell
+   METRICS_OPERATOR_IMAGE=ghcr.io/keptn/metrics-operator:v0.8.2
+   PATH_TO_SLI=sli.yaml
+   KEPTN_PROVIDER_NAME=my-prometheus-provider
+   KEPTN_PROVIDER_NAMESPACE=keptn-lifecycle-poc
+
+   docker run -v .:/mydata $METRICS_OPERATOR_IMAGE \
+     --convert-sli=mydata/$PATH_TO_SLI \
+     --keptn-provider-name=$KEPTN_PROVIDER_NAME \
+     --keptn-provider-namespace=$KEPTN_PROVIDER_NAMESPACE > analysis-value-template.yaml
+   ```
+
+   This command creates an `AnalysisValueTemplate` resource
+   that is defined in a file called `analysis-value-template.yaml`.
+   You can apply this to your cluster with a command like the following.
+   Be sure to specify the namespace;
+   if you omit it, the yaml file is applied to the default namespace.
+
+   ```shell
+   oc apply -f analysis-value-template.yaml -n keptn-lifecycle-poc
```

Comment:
My apologies for forgetting to adapt this beforehand, but the draft on which this guide is based on was made for an OpenShift user, where `oc` is the equivalent of `kubectl`. To stay as general as possible, let's use ` kubectl` in this guide here.

```suggestion
   kubectl apply -f analysis-value-template.yaml -n keptn-lifecycle-poc
```